koans/tic-tac-toe/README.md

```diff
@@ -0,0 +1,30 @@
+# Tic Tac Toe
+
+This is an exercise in Software Design and Changing Requirements.
+
+Don't peek ahead at the later steps of this koan, as part of the point of this is to deal with changing requirements.
+
+This Koan is written assuming you will be using Ruby.
+
+## Requirements
+
+Using the discipline of TDD, build a game of tic tac toe.
+
+* The game should allow a human to play against an AI.
+* The AI should be unbeatable.
```

Comment:
Can i recommend this is in a separate part? It was the sticking point for most people in LearnTech and it might make people feel a bit demotivated if they struggle with the first part. Getting it done with just 2 players is a good first step IMO

Reply:
Don't forget they will have a significant amount more time to spend on this...

Reply:
It's not necessarily the time that I'd be concerned about, more that I'd view the TDD-ing of that particular solution a problem in itself that can also be a "change of requirements", i.e. have they built it so the players could be swapped out. 

I think the actual challenge should be in there, but it was the part that I think tripped a lot of others up and could make part one feel not great. I would personally find that if I was struggling with part one of the problem that I'd be less motivated to continue

Reply:
Then again, that could just be me

Reply:
You know I see the emotive response to be a strength of the exercise. Emotive feelings will create a stronger driver to avoiding that pain in future.